src/shared/components/bottomSheet/BottomSheetWrapper.tsx

```
@@ -0,0 +1,114 @@
+    });
+
+  return (
+    <>
+      <div
+        className={clsx(styles.backdrop, isOpen && styles.backdropVisible)}
+        onClick={closeWithAnimation}
+        onTouchStart={(e) => {
+          e.stopPropagation();
+          closeWithAnimation();
+        }}
+      />
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)